db/migrate/20220810000616_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
* Kindly reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships. as all foreign keys should match what we have in the `ERD` Diagram 
![image](https://user-images.githubusercontent.com/52497246/183892339-955efc64-68f9-4baf-82c3-cdecdb595289.png)